pkg/services/blockfetcher/blockfetcher.go

```diff
@@ -539,9 +507,9 @@ func (bfs *Service) retry(action func() error) error {
 		case <-bfs.ctx.Done():
 			return bfs.ctx.Err()
 		}
-		backoff *= time.Duration(backoffFactor)
-		if backoff > maxBackoff {
-			backoff = maxBackoff
+		backoff *= time.Duration(neofs.BackoffFactor)
+		if backoff > neofs.MaxBackoff {
+			backoff = neofs.MaxBackoff
 		}
 	}
 	return err
```

Comment:
use neofs.URIScheme in func (bfs *Service) objectGet(ctx context.Context, oid string)

Reply:
Just implement it, I'll review afterwards.

Reply:
done